src/Security/Voter/PelagosEntityVoter.php

```diff
@@ -29,6 +29,7 @@ abstract class PelagosEntityVoter extends Voter
 {
     const CAN_CREATE = 'CAN_CREATE';
     const CAN_EDIT = 'CAN_EDIT';
+    const CAN_READ   = 'CAN_READ';
```

Comment:
Everyone can always read, we probably don't have to vote on that.

Reply:
I was thinking of restricted mode later down the line, but yeah, we can implement that when we need to.